src/framework/core/js/FluidView.js

```diff
@@ -119,6 +120,9 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             fluid.fail("fluid.container was supplied a non-jQueryable element");
         }
 
+        container.selector = selector;
```

Comment:
We should add some explanation here of what effect this logic is meant to achieve - perhaps in the form of a JIRA. It should explain that the fidelity of this approach in recovering the selector is limited (especially if the node comes from outside the system) and that this can only be depended on in limited contexts. Should explicitly mention the code in the prefs framework which relies on this as it tries to cross the iframe boundary in separatedPanel.